data-prepper-plugins/aws-lambda/src/main/java/org/opensearch/dataprepper/plugins/lambda/processor/LambdaProcessor.java

```diff
@@ -211,28 +216,19 @@ List<Record<Event>> convertLambdaResponseToEvent(Buffer flushedBuffer,
         List<Event> parsedEvents = new ArrayList<>();
 
         SdkBytes payload = lambdaResponse.payload();
-        // Handle null or empty payload
-        if (payload == null || payload.asByteArray().length == 0) {
-            LOG.warn(NOISY,
-                    "Lambda response payload is null or empty, dropping the original events");
-            return responseStrategy.handleEvents(parsedEvents, originalRecords);
+        // Considering "null" payload as empty response from lambda and not parsing it.
+        if (!("null".equals(payload.asUtf8String()))) {
```

Comment:
Can you explain the `"null"` string situation?

Reply:
Like then Lambda response is a plain one word "null" then we are considering it as empty response and not trying to parse it. It is basically like no response from lambda so we are going to drop all the records in the case of AggregateMode but fail in the case of StrictMode

Reply:
Is this a Lambda response or a response from the Lambda function?

Reply:
This is lambda response.

Reply:
Let's make this a constant with a useful name.

e.g.

```
private static final NO_RETURN_RESPONSE = "null"
```